docs/markdown/Cross-compilation.md

```diff
@@ -82,8 +102,8 @@ of a wrapper, these lines are all you need to write. Meson will
 automatically use the given wrapper when it needs to run host
 binaries. This happens e.g. when running the project's test suite.
 
-The next section lists properties of the cross compiler and thus of
-the host system. It looks like this:
+The next section lists properties of the cross compiler and its target
+system, and thus of our host system. It looks like this:
```

Comment:
'our' could perhaps be taken to mean 'of meson', so perhaps something like 'the host system of things built with it [the cross-compiler]`

Reply:
good point! will elaborate